impress-org/givewp
# Fix: Apply more escaping to prevent XSS vulnerabilities

## Description

This is a follow up to #6438 with even more fixes in place. Namely, this targets places where the URL is omitted, which fell back on `$_SERVER['REQUEST_URI']`, which can be considered vulnerable.

This is all related to the following wp.org effort: https://make.wordpress.org/plugins/2015/04/20/fixing-add_query_arg-and-remove_query_arg-usage/

## Affects

Large portions of the admin-side such as:
- Imports
- Exports
- Donor actions
- Donor registration emails
- Form registration
- Emails
- Stripe account onboarding

## Testing Instructions

Test out the following clear examples:
- `https://example.com/wp-admin/edit.php?post_type=give_forms&page=give-tools&a"><script>alert(/XSS/)</script>`
- `https://example.com/wp-admin/edit.php?post_type=give_forms&page=give-tools&tab=import&a"><script>alert(/XSS/)</script>`

Mostly, make sure that the affected areas continue to work without issue.

## Pre-review Checklist
